src/main/java/athena/task/taskfilter/ForecastFilter.java

```diff
@@ -17,6 +17,9 @@ public ForecastFilter(Forecast forecast) {
         this.forecast = forecast;
     }
 
+    public void setTodayDate(LocalDate todayDate) {
+        this.todayDate = todayDate;
+    }
```

Comment:
This sounds like it's for setting today's date. It looks like the intention is to change the date to be filtered. I think we should make a new `Forecast` value for this purpose instead of reusing `TODAY` because it can be quite confusing. In that case, I find `filterDate` to be clearer than `todayDate` too.